rpc/client/rpc_test.go

```diff
@@ -558,48 +542,64 @@ func TestTxSearch(t *testing.T) {
 		require.Len(t, result.Txs, 0)
 
 		// check sorting
-		result, err = c.TxSearch("tx.height >= 1", false, nil, nil, "asc")
```

Comment:
so in newer versions of Tendermint, it won't be possible to search for transactions based on height by default. It will break a lot of clients. How app can tell Tendermint to index event by height?

Reply:
There are two options here:

1) expose a way to define the key (not preferred by me)
2) default index txs by height when the KVIndexer is set
   I think this one makes more sense because if someone who does not want to index, the will turn off the indexer instead of just leaving it around

Reply:
👍 for 2

Reply:
will do if event.Index || key == tx.Height then the applicatioin can set it